lualib/resty/events/worker.lua

```diff
@@ -75,13 +94,15 @@ local function start_timer(self, delay)
     end))
 end
 
+local function terminate(self)
```

Comment:
Should we rename it to `terminating` or `terminated`?

Reply:
@chronolaw yes.